src/ut/cache_test.cpp

```diff
@@ -42,12 +42,16 @@
 #include "test_interposer.hpp"
 #include "fakelogger.h"
 
+#include "mockhttpresolver.h"
+#include "fake_cassandra_connection_pool.h"
 #include "mock_cassandra_store.h"
 #include "mockcommunicationmonitor.h"
 #include "cass_test_utils.h"
 
 #include <cache.h>
 
+typedef MockHttpResolver MockCassandraResolver;
```

Comment:
Ewww. I think the mocks should mirror what we're doing with the real classes (e.g. if we have an ARecordResolver and two typedefs, we should also have a MockARecordResolver and two additional typedefs).